src/pages/Inquiry/Inquiry/components/InquiryList/index.tsx

```diff
@@ -6,85 +6,113 @@ import {
   useState,
 } from 'react';
 
-import { InquiryContent } from 'api/inquiry/entity';
-import InquiryBlock from 'pages/Inquiry/Inquiry/components/InquiryList/components/InquiryBlock';
-import useInquiryList from 'pages/Inquiry/Inquiry/hooks/useInquiryList';
+import { InquiryContent, InquiryProps } from 'api/inquiry/entity';
+import useGetInquiry from 'pages/Inquiry/hooks/useGetInquiry';
+import InquiryBlock from 'pages/Inquiry/Inquiry/components/InquiryList/InquiryBlock';
 import cn from 'utils/ts/classNames';
 
 import styles from './InquiryList.module.scss';
 
+const RENDER_SIZE = 10;
+
 interface InquiryListProps {
   className: string;
-  typePath: string;
+  queryType: string;
```

Comment:
요런식으로 작성해주면 코드를 파악하기 용이할것같아여
```suggestion
  queryType: '내가쓴글' | '전체보기'
```

Reply:
타입을 유의하여 리팩토링하였습니다!
전체, 자신, 검색에 따라 페이지를 한데 아우르는 컴포넌트에서 타입 단언이 필요한 부분은 단언하고 삭제하거나 재사용할 타입도 리팩토링하였습니다.
9aa080c9ebb8a1f118607db2e4d01870b02ed3c2